src/odemis/driver/test/npmc_test.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+Created on Jun 21, 2018
+
+@author: Anders Muskens
+
+Copyright © 2018 Anders Muskens, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+
+from __future__ import division
+
+import logging
+from odemis.driver import npmc
+import odemis.model as model
+from odemis.util import test
+import os
+import time
+import unittest
+
+logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+logging.basicConfig(filename='test_npmc.log', level=logging.DEBUG)
+
+# Export TEST_NOHW=1 to force using only the simulator and skipping test cases
+# needing real hardware
+TEST_NOHW = (os.environ.get("TEST_NOHW", 0) != 0)  # Default to Hw testing
+
+C = 2.99792458e11  # speed of light in mm/s
+
+# arguments used for the creation of basic components
+CONFIG = {"name": "Delay Stage",
+          "role": "delay-stage",
+          "port": "/dev/ttyUSB0",
+          "axes": {
+                'x': {
+                    'number': 1,
+                    'range': [-1, 1],
+                    'unit': 'm',
+                },
+           },
+}
+
+COMP_ARGS = {
+    "atol": 1e-2,
```

Comment:
If the moves are in "m", the absolute value is awfully big (+- 1 cm!). Even if they are in mm, as the moves are in the order of "0.1", that's still 10%. I'd suggest to go down to 1e-3, at least.